src/mapper/github.mapping.ts

```diff
@@ -0,0 +1,31 @@
+export function mapProject(
+  repository: any,
+  languages: string[],
+  contributors: any[]
```

Comment:
i fail to understand why does this method have this interface. why not  `mapProject(project: Project)`

and also `export function mapProjects(projects: []Project)` ?

Reply:
@umsu2 

As we are dealing with different endpoints for Languages and contributors, we decided to provide the mentioned interfaces. Languages' variable includes a list of all used languages in the repository and contributors consist of an object with Username and Avatar URL.